internal/cmd/suggest.go

```diff
@@ -38,7 +38,13 @@ bad. Please use with discretion.
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := graphql.ContextWithTrackInput(cmd.Context(), trackInputFromCmd(cmd, args))
 
-			client, err := graphql.New(graphqlEndpoint(), newAPIClient(cmd.Context()))
+			token, err := newAuth().GetToken(cmd.Context())
+			if err != nil {
+				if err := checkAuthenticated(ctx, cmd, !recoverableWithLogin(err)); err != nil {
+					return err
+				}
+			}
+			client, err := graphql.New(graphqlEndpoint(), newAPIClient(cmd.Context(), token))
```

Comment:
Ok, changes implemented.

Reply:
> Note that this would need to be passed to `checkAuthenticated` as well.

We should also pass the auth object to `checkAuthenticated` since it calls `login` on an auth object @duvanmonsa